src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.scss

```diff
@@ -51,4 +51,8 @@
             color: $bdl-gray-62;
         }
     }
+
+    .bcs-AnnotationActivity-commentEditor {
```

Comment:
There are some styles on line 40 related to the comment form, including `border-top: none`. Are those no longer needed?

Reply:
Good catch - I need those as well.

Reply:
We've duplicated the styles from line 40. Are those styles still used? If so, where? If not, should they be removed? Can we reuse the existing class instead of creating a new one?

Reply:
We can reuse the existing class.